### ISSUE TITLE
Strategy Memo: Beeminder Is For Nerds

### ISSUE THREAD

**Issue by dreeves** [OneGraph/essay.dev#7]
<img class="aligncenter"
alt="Beeminder logo wearing nerd glasses and a pocket protector"
width="450px" height="450px"
src="https://user-images.githubusercontent.com/476818/99445374-4fae5380-28d2-11eb-80ff-7290ede1a3a5.png" />

> *This is crossposted at [blog.beeminder.com/nerds](https://blog.beeminder.com/nerds/ "I.e., the Beeminder blog").*

For years we've gotten advice to [widen our appeal](
  https://forum.beeminder.com/t/on-making-beeminder-more-appealing-to-a-wider-audience/5246
  "Forum post: On making Beeminder more appealing to a wider audience. It's very good and not actually nearly as at odds with this blog post as it appears."
). We shall now explain why you're all wrong[^1].

Let's start with an intuition-shaping factoid: GitHub is focused 100% on developers even though writers and designers and many other categories of people could be -- ought to be! -- using version control. (Additional factoid: GitHub was worth $7.5 billion when Microsoft bought them, with tens of millions of users -- all programmers.)

And Facebook, despite all its mainstream appeal now, started out focused exclusively on college students. Harvard students, specifically. In fact, their early growth strategy was to very carefully and deliberately add a single university at a time and only much later expand beyond university students.

Or take [Habitica](
  https://habitica.com
  "One of Beeminder's BFFs and integration partners"
) -- bigger than Beeminder while catering strictly to people so nerdy they're into RPGs. Or Discord, a perfectly general chat app which got huge by focusing on gamers. They have like a quarter billion users and only relatively recently showed any interest in anyone but gamers.

Running with the GitHub-is-for-devs example, you can imagine how they must make all their design and strategic decisions: "What works best for Daphne the Developer?" They're not afraid to scare off novelists who don't know programming lingo. GitHub is brilliant for novelists and there's nothing stopping them using it, but that's not who GitHub is _for_. There's a lot of value in only having one kind of user to worry about. Product design and making users happy is hard and gets combinatorially harder the more kinds of users and use cases you have to support.

<h4 class="pullquote">"Beeminder is probably not for you"</h4>

For Beeminder, we make all decisions from the perspective of what's best for Quentin the [Quantified-Self akratic ambitious self-aware high-integrity lifehacking graph-loving data nerd](
  https://blog.beeminder.com/typebee/ 
  "Type Bee Personality"
). Also he loves puns. If none of that speaks to you then, for now, Beeminder is probably not for you. We don't want to actively push you away, we just need to only be thinking about our core demographic.

We should emphasize that there are plenty of issues with Beeminder's usability to this core demographic; more on that below. Also, I'm exaggerating how narrow our core demographic is. "Data nerds" probably suffices.

### Isn't that placing an unnecessary fence around expansion?

Counterintuitively, it's the opposite! Paul Graham explains it pretty well in the "Well" section of his [essay on startup ideas](
  http://www.paulgraham.com/startupideas.html
  "'How to Get Startup Ideas (2012)"
). He describes two types of startups: those that a large number of people need a little, and those that a small number of people need a lot. (He also explains why startups can't be the best of both, and why he calls the second type "digging a deep well".)

> Nearly all good startup ideas are of the second type. Microsoft was a well when they made Altair Basic. There were only a couple thousand Altair owners, but without this software they were programming in machine language. [...]
> 
> In practice the link between depth and narrowness is so strong that it's a good sign when you know that an idea will appeal strongly to a specific group or type of user.

Or here's Patrick McKenzie of Stripe in [an old Twitter thread](
  https://twitter.com/patio11/status/1260240520778969088
  "I sure wish patio11 would use a different platform than Twitter"
) explaining why the book "Writing for Software Developers" is focused on software developers when there seems to be no reason for it not to be way more general:

> It's an artifact for the consumption of software developers which includes basically no software.
>
> It's a book about writing which intentionally anti-targets 99.99% of writers.
> 
> Why? Because constraining the topic to the needs of software developers specifically makes *the artifact much better* (you know you're writing for someone who cares about e.g. blog posts to get senior developer jobs and not midlist fiction, and can tailor advice appropriately).
> 
> And note that you certainly could have hypothetically written Writing For $PROFESSION for literally any profession but choosing Software Developers means you structurally have a client who has effectively infinite budget for books which deliver career effectiveness.
> 
> This lets you \*clears throat\* Charge More.

Or here's [a different old Twitter thread](
  https://twitter.com/patio11/status/1102447292177559552
  "I sure wish patio11 would use a different platform than Twitter"
) where he's talking about businesses rather than books, though it's all the same lesson:

> When I was consulting life got better in every way after I said “Eff it: B2B SaaS businesses, $10M to $50M in annual revenue, probably engineer-led” as the target market. Pitches got crisper. Results got better. Rates went up.

### What does this mean we should do?

It's mostly about what we shouldn't do. Like if we're, say, explaining a hypothetical new feature to show a non-cumulative version of graphs, we can call it a derivative and not bother to define what a derivative is[^2]. In other words, we should do what comes naturally to us. When talking amongst ourselves, it wouldn't occur to us define terms like "derivative". So the main thing to do is not hold back -- let our freak flags fly!

At this point, when this was just an internal memo, I emphasized that I was less certain about this than I sounded. Now it seems almost obvious. But also it matters less than it sounds in terms of Beeminder's roadmap. Regardless of the degree to which we want to double down on nerds, there's a long list of things we need to do to make Beeminder better for nerds and normals alike. Things like lifecycle emails,  smoothing onboarding friction, getting confusing advanced shıt out of sight of newbees, fixing the miasma of brokenness with scheduled breaks / ratchets / weekends-off / restarts, interactive graphs and [road.beeminder.com/tutorial](
  https://road.beeminder.com/tutorial
  "Huge thanks to Uluc for making this; I'm eager to get it in front of more newbees!"
) and [road.beeminder.com/sandbox](
  https://road.beeminder.com/sandbox
  "This one is less newbee-friendly but is pretty powerful!"
). The list goes on and on and there's a huge newbee focus in most of it.

When normal humans can pass [hallway tests](
  https://www.joelonsoftware.com/2000/08/09/the-joel-test-12-steps-to-better-code/
  "AKA informal usability tests -- see #12 in The Joel Test"
) -- and it's critical that they be able to! -- only then, at the earliest, should we think about adjusting copy and marketing and UI and everything to target them. In the meantime, almost all our onboarding and newbee-related problems have best-of-both-worlds solutions. We're not backing ourselves into any corners (not more so than we already are, at least) with our nerd focus. The work ahead of us, at least for some months, involves improving onboarding and reducing confusion in ways that applies equally to nerds and normals.

At some point we'll find ourselves deciding between choice A that's better for QS-y Quentin and choice B that's better for Normal Norman and that's when we can revisit this. (But probably the answer will be a resounding "Choice A! Norman can suck it" for the foreseeable future.)

### Other benefits of focusing on nerds

First of all, it helps our own focus, our clarity, even our motivation to be helping fellow nerds. Relatedly, our distinctive voice and vibe and culture is pretty key to us having True Fans (which holy cow do we ever have, and it feels amazing). Next, being a bit exclusionary -- even alienating users who aren't quite your target audience -- attracts and endears and generates passion among those who are. They feel part of an in-group.

As a bonus, nerds tend to be easier and more valuable in support. Maybe sometimes they can be a bit much, but very positive on net, we've found.

#### Troll filtering and expectations management

It's infuriating when people leave low-effort 1-star reviews in the app store. Sometimes they do that without even logging in and there's little we can do about that. But -- and this one is more conjecture so far -- making it obvious that Beeminder is for nerds (equations on the splash screen, we're not sure yet) will tend to make those trolls' eyes glaze over and not bother trolling.

Or imagine someone who's like "Goals? I have goals!" and starts signing up and then sees graphs and numbers and is like "WTF? 1 star." The more front-and-center the graphs and numbers are, the less likely they are to feel frustrated and disappointed.

#### Anti-example

Take our oldest competitor, [StickK.com](
  https://stickk.com
  "We feel bad being this derogatory toward our once-esteemed competitor but, well, StickK has been stuck in the mud for years, like almost literally. The founders have all left."
), targetting anyone with goals. Or you could say they target anyone who's [akratic](
  https://blog.beeminder.com/akrasia
  "Inaugural post on the Beeminder blog. It's not a decade out of date, it's a classic!"
) -- those who struggle to stick to their intentions -- but that's still far too broad a swathe of the population. They're clearly trying to have mainstream appeal and they just end up being, well, [StickKly](
  https://stickk.com
  "We feel bad being this derogatory toward our once-esteemed competitor but, well, StickK has been stuck in the mud for years, like almost literally. The founders have all left."
). 

<br>&nbsp;<br>

On that slightly meanspirited note, we rest our case. Broadening our appeal too soon is at least one classic startup mistake we've avoided. We can [cross the chasm](
  https://en.wikipedia.org/wiki/Crossing_the_Chasm
  "Classic business book about 'Marketing and Selling High-Tech Products to Mainstream Customers'"
) in due time.

<br>&nbsp;<br>

[^1]: Actually there's an interesting selection bias here. The thesis of this post is very standard startup wisdom. Those who know it don't think to mention it to us. After all, we seem to be doing it. (Also no one thinks to mention to us that we're smart to have an API or to have smartphone apps or to, I don't know, label the axes on our graphs. Also [everything is obvious once you know the answer](https://www.amazon.com/dp/B00556G7RY/ref=dp-kindle-redirect?_encoding=UTF8&btkr=1 "Book by Duncan Watts").) So we don't tend to hear from those who agree that Beeminder should focus on nerds. In any case, we do change our minds aplenty and have done plenty of 180s based on feedback and advice, but in this case, after a lot of thought, we've decided to double down on our initial instincts.

[^2]: To clarify, not remembering calculus emphatically does not make someone dumb! We, as Beeminder, just take it as a background assumption that you're conversant in that kind of language. We're always delighted to clarify and explain if you ask!

*Image credit: [Faire Soule-Reeves](
  http://dreev.es/eli5
  "Faire doing a Beeminder explain-like-I'm-5 with her then 5-year-old brother"
)*

```backmatter
{"canonical": "https://blog.beeminder.com/nerds/"}
```

**Comment by sgrove** [comment]
Heeeey!

**Comment by sgrove** [comment]
Heeeey!
